bower.json

```diff
@@ -5,10 +5,16 @@
     "dist/rollbar.umd.js"
   ],
   "ignore": [
+    "dist/*.snippet*",
```

Comment:
I was trying to update, but the example is now outdated: https://github.com/rollbar/rollbar.js/tree/master/examples/bower#using-rollbar-with-bower
Should we update the instructions, or keep delivering the snippet?